examples/visualization/plot_evoked_arrowmap.py

```diff
@@ -0,0 +1,30 @@
+"""
+========================================
+Plotting topographic arrowmaps of evoked data
+========================================
+
+Load evoked data and plot topomaps for selected time points.
+"""
+# Authors: Sheraz Khan <sheraz@khansheraz.com>
+#
+# License: BSD (3-clause)
+
+
+from mne.datasets import sample
+from mne import read_evokeds
+from mne.viz.topomap import plot_arrowmap
```

Comment:
you should be able to import it from mne.viz

Reply:
Done